### ISSUE TITLE
Cannot search specifically for Grand Spectrum Mana / Armor / Elemental

### ISSUE THREAD

**Issue by superouman** [licoffe/POE-Stash-indexer-NG#15]
The three Grand Spectrum jewels data is mixed up and they can't be separated : http://poe-rates.com/index.php?league=Legacy&item=Grand%20Spectrum&interval=12h

**Comment by licoffe** [comment]
Thanks for reporting! There is currently no easy fix for this issue as different item types are not supposed to have the same unique name in the current design, but I'll let you know when it's done.

**Comment by ogier** [comment]
Some other items have similar issues. e.g. Vessel of Vinktar has three different effects.

**Comment by licoffe** [comment]
@ogier Yes, I wish I could do that, but this is not possible since I am not storing the mods in the indexer version I use for poe-rates.com
